src/OrchardCore.Modules/OrchardCore.Email/ViewModels/SmtpSettingsViewModel.cs

```diff
@@ -1,16 +1,13 @@
-using System;
-using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Localization;
 
 namespace OrchardCore.Email.ViewModels
 {
-    public class SmtpSettingsViewModel : IValidatableObject
```

Comment:
@MikeAlhayek this the proper way to handle the data attributes validation after localizations support